src/interface/updatechestinventory.cpp

```diff
@@ -78,9 +78,12 @@ int numItemsInChest()
 
 	int i = 0;
 
-	for (node = chestInventory->first; node != nullptr; node = node->next)
+	if ( chestInventory )
```

Comment:
Aha, was that a chest crash we never chased down?

Reply:
I never saw that one in action, but was definitely dangerous :D